Police-Data-Accessibility-Project/data-sources-app-v2
# feat: data source by id route

Route to view data source by its id.

## To test
1. run app locally
2. search for some data sources
3. Click on any of them
4. Ensure it matches the mocks [here](https://www.figma.com/design/UsbyOrgtBnLMyyqlfqIOqm/PDAP-v2?node-id=6385-3374&m=dev), more or less
